src/samples/nodejs/gpio_write.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This sample code blinks an LED connected to Edison GPIO pin 48.
+ */
+var gpio = require( "soletta/gpio" ),
+    ledPin = null,
+    state = false;
+
+// Change the LED state
+function changeState() {
+    state = !state;
+    ledPin.write( state ).then( function() {
+        console.log( "LED state changed" );
+    } ).catch( function( error ) {
+        console.log( "Failed to write on GPIO device: ", error );
+        process.exit();
+    } );
+
+    setTimeout( changeState, 1000 );
+}
+
+// Configure LED pin.
+gpio.open( {
+    pin: 48
+} ).then( function( pin ) {
+    ledPin = pin; // Save the handle
+    changeState(); // Change the LED state.
+} ).catch( function( error ) {
+    console.log( "Could not open GPIO: ", error );
+    process.exit();
+} );
+
+process.on( "SIGINT", function() {
```

Comment:
I'd guess this is for 'Ctrl + C' to work, right? State that. I shall update my sample soon, as well.